packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -350,46 +350,49 @@ curl -v -X DELETE \
 "https://${yourOktaDomain}/api/v1/groups/00gsr2IepS8YhHRFf0g3/roles/grasraHPx7i79ajaJ0g3"
 ```
 
-###### Response Example
+##### Response example
 
 
 ``` http
 HTTP/1.1 204 No Content
 ```
 
-## Role Target Operations
+## Role target operations
 
-### Group Administrator Role Group Targets
+Role targets are a way of defining permissions for admin roles into a smaller subset of Groups or Apps within your org. Targets limit an admin's permissions to a targeted area of the org. You can define admin roles to target Groups, Applications, and Application Instances.
 
-#### List Group Targets for Group Administrator Role
+### Group administrator role Group targets
 
-##### List Group Targets for Group Administrator Role given to a User
+Assigns a Group admin role to a specific Group that grants the admin permission to manage only that Group. For example, an admin role may be assigned to manage only the IT group. The permissions for specifically what an admin can do within that Group depends on the admin role that they are assigned to. See [Administrators](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Administrators).
+
+#### List Group targets for the Group administrator role
+
+##### List Group targets for the Group administrator role given to a User
 
 
 <ApiOperation method="get" url="/api/v1/users/${userId}/roles/${roleId}/targets/groups" />
 
-Lists all group targets for a `USER_ADMIN` or `HELP_DESK_ADMIN` role assigned to a user.
+Lists all Group targets for a `USER_ADMIN` or `HELP_DESK_ADMIN` Role assigned to a User
 
-###### Request Parameters
+###### Request parameters
 
 
 | Parameter   | Description                                                    | Param Type   | DataType   | Required |
 | :---------- | :------------------------------------------------------------- | :----------- | :--------- | :------- |
-| userId      | `id` of a user                                                 | URL          | String     | TRUE     |
```

Comment:
For now, the clearest way to order things is alphabetically, since that gives you a way to quickly find a term that you're looking for. More generally, we shouldn't be mixing URL and Query params in a single table, but redoing all of our param tables would require eng-wide effort.